application.py

```diff
@@ -29,7 +32,23 @@ def index():
 def register():
     # # form = request.form
     if request.method == "POST":
-        print("Name: ",request.form['username'])
-        print("Passowrd: ",request.form['password'])
-        return "<h2 text-align=\"center\">Hello "+request.form['username']+", Welcome !</h2>"
```

Comment:
I know for now you have used it in this way. Next time use Html page for this kind..